machine_learning/losses/mean_squared_error.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
```

Comment:
Please include an explanation of MSE (along with a link to a source as well as the formula) in a header docstring